src/components/cluster/ClusterForm.tsx

```diff
@@ -181,6 +186,10 @@ export default function ClusterForm({
             prometheusTlsClientCert: {
                 required: false,
             },
+            proxyUrl: {
+                required: id && KubectlProxyCheckBox ? toConnectViaProxyTemp : false,
```

Comment:
(id && KubectlProxyCheckBox) && toConnectViaProxyTemp

Reply:
Are you expecting proxyUrl as a boolean ?

Reply:
done